modules/core/arrow-core/src/main/kotlin/arrow/core/extensions/either.kt

```diff
@@ -135,7 +130,7 @@ interface EitherApplicativeError<L> : ApplicativeError<EitherPartialOf<L>, L>, E
   override fun <A> raiseError(e: L): Either<L, A> = Left(e)
 
   override fun <A> EitherOf<L, A>.handleErrorWith(f: (L) -> EitherOf<L, A>): Either<L, A> =
-    fix().eitherHandleErrorWith(f)
+    fix().handleErrorWith(f)
```

Comment:
Same here, and in the others. Some of these are defined as extfuns already, so it picks up this one instead unless you alias it.

Reply:
thank you I went crazy to find the problem

Reply:
There's only a couple like constAp where this doens't apply.